viam-orb-slam3_test.go

```diff
@@ -548,100 +477,18 @@ func TestGeneralNew(t *testing.T) {
 	t.Run("New slam service with bad camera", func(t *testing.T) {
 		attrCfg := &slamConfig.AttrConfig{
 			Sensors:       []string{"gibberish"},
-			ConfigParams:  map[string]string{"mode": "2d"},
+			ConfigParams:  map[string]string{"mode": "mono"},
 			DataDirectory: name,
 			DataRateMsec:  validDataRateMS,
 			UseLiveData:   &_true,
 		}
 
 		// Create slam service
-		_, err := createSLAMService(t, attrCfg, "fake_cartographer", logger, false, false)
+		_, err := createSLAMService(t, attrCfg, logger, false, false)
 		test.That(t, err, test.ShouldBeError,
 			errors.New("configuring camera error: error getting camera gibberish for slam service: \"gibberish\" missing from dependencies"))
 	})
 
-	t.Run("New slam service with invalid slam algo type", func(t *testing.T) {
```

Comment:
This test was removed because it is designed to test SlamLibraries which we no longer use.